pkg/ecdsa/retry_test.go

```diff
@@ -0,0 +1,106 @@
+package ecdsa
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/keep-network/keep-core/pkg/chain"
+)
+
+func isSubset(groupMembers []chain.Address, subset []chain.Address, t *testing.T) {
+	memberMap := make(map[chain.Address]struct{})
+	for _, operator := range groupMembers {
+		memberMap[operator] = struct{}{}
+	}
+	for _, operator := range subset {
+		if _, ok := memberMap[operator]; !ok {
+			t.Errorf("Subset member [%s] is not in the operator group.", operator)
+		}
+	}
+}
+
+func testEq(a, b []chain.Address) bool {
```

Comment:
You can do the same using the built-in `reflect.DeepEqual` function.

Reply:
I was reading on https://stackoverflow.com/questions/15311969/checking-the-equality-of-two-slices that a `testEq` method like what I wrote is *wildly* faster than `reflect.DeepEqual`, though if we don't care, I don't care! It's test code, so it's not like it has to be performant

Reply:
Up to you! I'm good with both.

Reply:
```
== testEq ==
❯ go test
PASS
ok  	github.com/keep-network/keep-core/pkg/ecdsa/retry	0.097s

== reflect.DeepEqual ==
~/keep-core/pkg/ecdsa/retry retry-signing
❯ go test
PASS
ok  	github.com/keep-network/keep-core/pkg/ecdsa/retry	0.113s
```

`0.016s` is not worth the extra lines of code. Changing!

Reply:
[7feb73e](https://github.com/keep-network/keep-core/pull/3109/commits/7feb73ef2c3edbf07cfb313a8edc362fc41131b9
)